tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -65,6 +66,8 @@
 cmontecarlo_filepath = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
 cmontecarlo_methods = CDLL(cmontecarlo_filepath)
 
+INVERSE_C = 3.33564095198152e-11
```

Comment:
Sure but when I did that the tests failed so I hardcoded the value from the C part